src/index.html

```diff
@@ -12,6 +12,14 @@
       <link rel="stylesheet" type="text/css" href="/styles/noscript.css" />
     </noscript>
 
+    <meta name="defaultLanguage" content="en-US">
+    <meta name="availableLanguages" content="en-US, pl">
+    <link rel="localization" href="/locale/{locale}/dictionary.ftl">
+
+    <script src="/scripts/browser.js"></script>
```

Comment:
Is there a way to include these in the single JS file instead of having multiple dist files? It seems that transpiling the l20n code using babel could still be done as a prerequisite step to compiling the rest of the coffeescript code.